src/bootstrap/test.rs

```diff
@@ -352,6 +352,60 @@ impl Step for Rls {
     }
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct RustAnalyzer {
+    stage: u32,
+    host: TargetSelection,
+}
+
+impl Step for RustAnalyzer {
+    type Output = ();
+    const ONLY_HOSTS: bool = true;
```

Comment:
I think this is why CI didn't fail - this step doesn't run by default, only if you set `const DEFAULT: bool = true;`. Let's go ahead and do that now so you can get feedback from CI without having to run tests locally. (alternatively you could add a line to one of the dockerfiles, but I don't think that's a better approach.)
```suggestion
    const ONLY_HOSTS: bool = true;
    const DEFAULT: bool = true;
```